sklearn/metrics/tests/test_classification.py

```diff
@@ -197,6 +197,13 @@ def test_precision_recall_f_extra_labels():
         assert_raises(ValueError, recall_score, y_true_bin, y_pred_bin,
                       labels=np.arange(-1, 4), average=average)
 
+    y_true = np.array([[0, 1, 1], [1, 0, 0]])
```

Comment:
Because it's hard to see what this is testing, it would be good to add a comment saying that it tests non-regression on issue #xxx.

Reply:
There isn't any issue for this, should I instead refer this PR itself?